src/transaction/components/Transaction.tsx

```diff
@@ -4,6 +4,7 @@ import type { TransactionReact } from '../types';
 import { TransactionProvider } from './TransactionProvider';
 
 export function Transaction({
+  calls,
```

Comment:
Thinking out loud, `calls` for sure is the best prop to move forward for this. But I am curios if Wagmi long-term direction on this.